apps/qaqc/bash/droplet_setup.sh

```diff
@@ -0,0 +1,6 @@
+doctl compute droplet create \
```

Comment:
This file is not used, nor did I actually run it, but this is the command that would recreate the droplet with doctl (cli for DO) set up. Not sure if we feel like this would be worth having in here but included for now